src/components/settings/app.tsx

```diff
@@ -31,7 +32,8 @@ class AppTab extends React.Component<AppTabProps, AppTabState> {
             themeSettings: getThemeSettings(),
             itemSize: null,
             cacheSize: null,
-            deleteIndex: null
+            deleteIndex: null,
+            iconStatus: window.settings.getIconStatus()
```

Comment:
Could we use a better variable name here and below for `iconStatus` that includes "youtube" to make this clearer?